packages/blockchain-wallet-v4-frontend/src/scenes/Login/Wallet/TwoFA/index.tsx

```diff
@@ -68,6 +69,27 @@ const TwoFAWallet = (props: Props) => {
     authActions.resendSmsCode({ email, guid: formValues?.guid })
   }
 
+  const getTwoFaType = (): string | null => {
```

Comment:
Can we move this logic into a separate util function? It seems to be repeated across two components.

Reply:
I think it could also be used in [your other PR](https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/5561) too.